test/tla/cli-integration-tests.md

```diff
@@ -3414,7 +3414,7 @@ $ rm -rf ./test-out-dir ./test-run-dir
 ### configuration management: read run-dir from local `.apalache.cfg`
 
 ```sh
-$ echo "run-dir: ./configured-run-dir" > .apalache.cfg
+$ echo "common.run-dir: ./configured-run-dir" > .apalache.cfg
```

Comment:
Note that this introduces a change to the configuration format, which moves al the previous top-level config values under the `common` key.